docs/book/src/appendix/keywords.md

```diff
@@ -0,0 +1,88 @@
+# [Appendix A: Keywords](#appendix-a-keywords)
```

Comment:
You can remove the link here and for other headers, as they are automatically applied.